src/ORM/FieldType/DBText.php

```diff
@@ -130,10 +139,17 @@ public function Summary($maxWords = 50, $add = false)
             $add = $this->defaultEllipsis();
         }
 
+        $summarySentenceSeparatorsConfig = $this->config()->summarySentenceSeparators;
+        $summarySentenceSeparatorsArr = [];
+        foreach ($summarySentenceSeparatorsConfig as $separator) {
+            array_push($summarySentenceSeparatorsArr, '\\' . $separator[0]);
```

Comment:
The backslash escaping is only needed for some (special) characters, not for everything, right?
If I wanted to split say on `r` or `x` characters, it would make them `\\r` etc.

Reply:
Good point, this would escape something like a literal `s` to be a `\s` (which has special meaning in regex of whitespace chars), so not only is this not the correct approach it will actually cause unexpected behaviours.

Reply:
I think we may add a comment above the config prop for the devs to provide the escaping if required and basically don't try to be smart during the use, because we can't assume whether the dev meant `r` vs `\r` or `.` vs `\.`

Reply:
The feature isn't to allow devs to provide arbitrary regex expressions, but to provide a list of chars to break the sentence on (eg: [.?!]).

I think relying on devs to provide escaped regex is an overly fragile.

If someone wants to split on regex then they should probably implement that themselves or we might be better just providing a split function that takes regex.

Reply:
Actually yeah, if we list all of them within the `[]` then it's just a list of characters and even a `.` wouldn't be treated as special.